ci/kind/kind-setup.sh

```diff
@@ -242,6 +244,32 @@ function configure_extra_networks {
   done
 }
 
+function configure_kind_ipam_vlan_subnets {
+  if [[ -z $IPAM_VLAN_SUBNETS || -z $IPAM_VLAN_IDS ]]; then
+    return
+  fi
+  
+  IFS=',' read -r -a ipam_vlan_subnets <<< "$IPAM_VLAN_SUBNETS"
+  IFS=',' read -r -a ipam_vlan_ids <<< "$IPAM_VLAN_IDS"
+  i=0
+
+  for s in "${ipam_vlan_subnets[@]}"; do
+    vlan_id=${ipam_vlan_ids[$i]}
+    i=$(($i+1))
+    echo "Configuring VLAN-$vlan_id subnets"
+    bridge_id=$(docker network inspect kind -f {{.ID}})
```

Comment:
better to refactor retrieving bridge id get_bridge_id() {

    docker network inspect "$network_name" -f '{{.ID}}'

error handling
}
error handling should be considered to handle cases where the Docker network does not exist or the docker network inspect command fails for some reason.

Reply:
yes,added the cleaned up code